docs/ru/Quickstart-RU.md

```diff
@@ -0,0 +1,205 @@
+```
+
+Инструменты для:
+
+* Анализа и построения графа.
+* Получения сгенерированных классов
+
+```kotlin
+testImplementation("ru.kontur.mobile.visualfsm:visualfsm-tools:1.1.0")
+```
+
+## Как подключить и настроить кодогенерацию
+
+### Как подключить кодогенерацию
+
+#### _Для Котлин приложения_
+
+##### В gradle файле модуля, в котором будут использованы аннотации
+
+<details>
+  <summary>Groovy</summary>
+
+```groovy
+// Подключаем KSP плагин
+plugins {
+    id "com.google.devtools.ksp" version "1.6.21-1.0.6"
+}
+
+// Добавляем сгенерированный код в каталоги исходного кода
+kotlin {
+    sourceSets {
+        main.kotlin.srcDirs += 'build/generated/ksp/main/kotlin'
+        test.kotlin.srcDirs += 'build/generated/ksp/test/kotlin'
+    }
+}
+
+dependencies {
+    // Подключаем AnnotationProcessor
+    ksp "ru.kontur.mobile.visualfsm:visualfsm-compiler:1.1.0"
+    // Поключаем инструменты для удобного получения сгенерированного кода. Только для jvm проектов.
+    implementation "ru.kontur.mobile.visualfsm:visualfsm-tools:1.1.0"
+}
+```
+
+</details>
+<details>
+  <summary>Kotlin</summary>
+
+```kotlin
+// Подключаем KSP плагин
+plugins {
+    id("com.google.devtools.ksp") version "1.6.10-1.0.6"
+}
+
+// Добавляем сгенерированный код в каталоги исходного кода
+kotlin {
+    sourceSets.main {
+        kotlin.srcDir("build/generated/ksp/main/kotlin")
+    }
+    sourceSets.test {
+        kotlin.srcDir("build/generated/ksp/test/kotlin")
+    }
+}
+
+dependencies {
+    // Подключаем AnnotationProcessor
+    ksp("ru.kontur.mobile.visualfsm:visualfsm-compiler:1.1.0")
+    // Поключаем инструменты для удобного получения сгенерированного кода. Только для jvm проектов.
+    implementation("ru.kontur.mobile.visualfsm:visualfsm-tools:1.1.0")
+}
+```
+
+</details>
+
+#### _Для Андроид приложения_
+
+##### В gradle файле модуля, в котором будут использованы аннотации
+
+<details>
+  <summary>Groovy</summary>
+
+```groovy
+// Подключаем KSP плагин
+plugins {
+    id "com.google.devtools.ksp" version "1.6.21-1.0.6"
+}
+
+dependencies {
+    // Подключаем AnnotationProcessor
+    ksp "ru.kontur.mobile.visualfsm:visualfsm-compiler:1.1.0"
+    // Поключаем инструменты для удобного получения сгенерированного кода.
+    implementation "ru.kontur.mobile.visualfsm:visualfsm-tools:1.1.0"
+}
+```
+
+</details>
+<details>
+  <summary>Kotlin</summary>
+
+```kotlin
+// Подключаем KSP плагин
+plugins {
+    id("com.google.devtools.ksp") version "1.6.10-1.0.6"
+}
+
+dependencies {
+    // Подключаем AnnotationProcessor
+    ksp("ru.kontur.mobile.visualfsm:visualfsm-compiler:1.1.0")
+    // Поключаем инструменты для удобного получения сгенерированного кода.
+    implementation("ru.kontur.mobile.visualfsm:visualfsm-tools:1.1.0")
+}
+```
+
+</details>
+
+##### В gradle файле app модуля
+
+<details>
+  <summary>Groovy</summary>
+
+```groovy
+// Добавляем сгенерированный код в каталоги исходного кода
+android {
```

Comment:
android {
    ...  
    applicationVariants.all { variant ->
        variant.sourceSets.java.each {
            it.srcDirs += "build/generated/ksp/${variant.name}/kotlin"
        }
    }
}